draft-ietf-scitt-architecture.md

```diff
@@ -401,16 +401,19 @@ This specification leaves implementation, encoding and documentation of Registra
 
 #### Mandatory Registration Checks
 
-During registration, a Transparency Service MUST, at a minimum, authenticate the Issuer of the Signed Statement by validating the COSE signature and checking the identity of the issuer against one of its currently configured trust anchors, using the `x5t` (34), `x5chain`(33) or `kid`(4) protected headers of the Signed Statement as hints.
-For instance, in order to authenticate X.509 Signed Statements, the Transparency Service MUST build and validate a complete certificate chain from the Issuer's certificate identified by `x5t`, to one of the root certificates most recently registered as a trust anchor of the Transparency Service.
+During registration, a Transparency Service MUST, at a minimum, syntactically check the Issuer of the Signed Statement by validating the COSE signature and checking the Issuer details are consistent with one of its supported trust anchors.
+The Issuer identity MUST be bound to the Signed Statement by including a commitment to it in the protected header.
+If the protected header includes multiple commitments, all those that are registered by the Transparency Service MUST be checked.
+
+For instance, when using X.509 Signed Statements, the Transparency Service MUST build and validate a complete certificate chain from the Issuer's certificate identified by `x5t`, to one of the root certificates most recently registered as a trust anchor of the Transparency Service.
```

Comment:
I think roots can be regular public keys in second cases, even when x509 is used... But it would be good to check with experts on this

Reply:
If that passes muster with expert review, then I would happily take suggested wording to that effect

Reply:
Agreed, a new term "commitment" is something we'd like to avoid
Can folks please review the [suggestion from the editors meeting for using "identifier'](https://github.com/ietf-wg-scitt/draft-ietf-scitt-architecture/pull/231/files#r1567497546). Please +1, -1, or propose an alternative on that suggestion, .

Reply:
Since we discussed "identifier" this morning, I committed the change. But, folks can still make suggestions for alternatives.